src/coreclr/vm/threads.cpp

```diff
@@ -7702,6 +7702,18 @@ InterpThreadContext* Thread::GetInterpThreadContext()
 
     return m_pInterpThreadContext;
 }
+
+extern "C" InterpThreadContext* STDCALL GetInterpThreadContextWithPossiblyMissingThread(Thread *pThread)
+{
+    LIMITED_METHOD_CONTRACT;
```

Comment:
```suggestion
    CONTRACTL
    {
        THROWS;
        if (GetThreadNULLOk()) {GC_TRIGGERS;} else {DISABLED(GC_NOTRIGGER);}
    }
    CONTRACTL_END;
```
This should have same contract as SetupThread.

`LIMITED_METHOD_CONTRACT` as alias NOTHROW, GC_NOTRIGGER. It is not the correct contract to use here.